app/src/main/java/org/sagebionetworks/research/mpower/tracking/recycler_view/SymptomsLoggingItemViewHolder.java

```diff
@@ -84,7 +98,7 @@ public void setContent(@NonNull SimpleTrackingItemConfig config) {
     }
 
     private void setLogObservers(@NonNull TrackingItem trackingItem) {
-        final LiveData<SymptomLog> logLiveData = this.viewModel.getLoggedElement(trackingItem.getIdentifier());
+        final LiveData<SymptomLog> logLiveData = symptomLoggingListener.getSymptomLogData(trackingItem);
```

Comment:
I think having your ViewHolder implement LifecycleOwner is a sign that you are packing too much logic and functionality into the class. As I said before, ViewHolder is supposed to be lightweight and hold the view and provide basically one function called onBindView(data) that takes in a data object and sets the UI to the appropriate state. This allows all the ViewHolders to be reliably recycled by the RecylerView without any leaking observers, state, etc.  

My suggestion would be to have the Fragment (the real LifecycleOwner) observe the live data, update the data model in the adapter, and then call recylerViewAdapter.notifyItemChanged(...). The notify functions will also automatically animate your RecyclerView which is something that isn't happening with your current implementation of these ViewHolders.